x/mint/module.go

```diff
@@ -104,19 +100,16 @@ func NewAppModule(
 	cdc codec.Codec,
 	keeper keeper.Keeper,
 	ak types.AccountKeeper,
-	ic types.InflationCalculationFn,
+	// This input is unused as of Cosmos SDK v0.53 and will be removed in a future release of the Cosmos SDK.
+	_ types.InflationCalculationFn,
```

Comment:
The only clean way I could think to do this was to keep this input to this function but not use it.  This could be confusing to users though.  It is more clear, but breaking to users if the argument is just removed alltogether